minutes/7-7-17

```diff
@@ -0,0 +1,20 @@
+### Kitty system
+- labels on kitty to differentiate
+- two kittys
+- prices for visitors
```

Comment:
maybe you should also write this in the README just cause the repository is open to everyone on FACN2 organization so they need to see this and read it :maple_leaf: